src/phase0/state_transition/mod.rs

```diff
@@ -46,6 +46,8 @@ pub enum Error {
     },
     #[error("overflow")]
     Overflow,
+    #[error("invalid attestation")]
+    InvalidAttestation,
```

Comment:
see above comment, i think we can drop this -- if you did need this here, i would want to consider it amongst the other variants as `InvalidAttestation` really just refers to an invalid att in the context of a bad block, not some error during epoch processing